doc/user/zebra.rst

```diff
@@ -544,15 +544,16 @@ Consequently, the VRF naming `Default` will be overriden by the new discovered
 namespace name `vrf0`.
 
 For those who don't use VRF backend with *Linux network namespace*, it is
-possible to statically configure and recompile FRR. By using `DFLT_VRF_NAME`
-definition in file `defaults.h`, it is possible to choose an alternate name for
-default VRF. Then, the default VRF naming will automatically be updated with the
-new name.
+possible to statically configure and recompile FRR. It is possible to choose an
+alternate name for default VRF. Then, the default VRF naming will automatically
+be updated with the new name. To illustrate, if you want to recompile with
+`global` value, use the following command:
 
 .. code-block:: linux
 
-   #define DFLT_VRF_NAME    "global"
+   ./configure --with-defaultvrfname=global
 
+More information about the option in :ref:`_frr-configuration`.
 
```

Comment:
Hi @qlyoung , I will need your help for this issue.
at line 556, I fall on the compilation error:
```
doc/user/zebra.rst:563: WARNING: undefined label: _frr-configuration (if the link has no caption the label must precede a section header)
```
I want to add a reference from doc/user/zebra.rst to doc/user/installation.rst, but i looks like the reference is outside and is not found.
any ideas on how to handle this ?